include/nlohmann/json.hpp

```diff
@@ -7086,6 +7086,45 @@ class basic_json
         return ptr.get_checked(this);
     }
 
+
+    /*!
+    @brief remove element from a JSON tree given a pointer
+
+    Removes elements from a JSON tree with the pointer value @ptr
```

Comment:
Please add a comment that when the root is deleted, the whole value will be `null`.